concepts/eventing/performance evaluation/nats-jetstream/results_prod_HA.md

```diff
@@ -0,0 +1,329 @@
+  Warning  FailedMount             97s    kubelet                  Unable to attach or mount volumes: unmounted volumes=[eventing-nats-js-pvc], unattached volumes=[config-volume pid kube-api-access-gn9xs eventing-nats-js-pvc]: timed out waiting for the condition
+```
+
+---
+
+## Test Scenario 2: K8s Cluster Node deleted during test where stream leader NATS pod was deployed
+
+### Run ID: 8/6/2022T14:24 (Duration: 5m, Event Rate: 150rps)
+
+> **NOTE:** Deleted k8s Node where `eventing-nats-2` Pod was deployed after 2 minutes of test execution.
+
+**State before test run:**
+- Stream:
+  - Leader: eventing-nats-2
+  - Replicas: 1
+
+**State after test run:**
+- Stream:
+  - Leader: NA (Stream Unavailable)
+  - Replicas: NA
+
+
+**Test Results:**
+```
+-> Total Events Sent      by **Test Sender** : 10,672 (+ 45 (Failed) = 10,717)
+-> Total Events Received  by **Sink**        : 10,672
+```
+
+**Test Results Dashboard:**
+
+![](assets/ha_crash2_1.png "")
+
+**Test Sender Stats:**
+
+![](assets/ha_crash2_2.png "")
+
+**Finding:** 
+
+Deleting the K8s Node where stream leader NATS Pod was deployed resulted in the JetStream stream to be unavailable as there was only one replica of the stream. NATS stopped accepting any new events for that steam and event-publisher-proxy was failing to publish any event to NATS. The system started to work again as soon as the crashed NATS Pod was deployed on another Node and stream became available again. Therefore, a single stream relpica is not enough to cater a situation of single-node failure.
+
+---
+
+## Test Scenario 3: NATS Pod (JetStream Stream Leader) deleted during test
+
+> **NOTE:** Deleted the pod `eventing-nats-1` (Stream Leader) after 3 minutes. Notice that this time only the pod was deleted, not the k8s Node itself.
+```
+kubectl delete po -n kyma-system eventing-nats-1
+```
+
+### Run ID: 8/6/2022T11:28 (Duration: 10m, Event Rate: 150rps)
+
+**State before test run:**
+- Stream:
+  - Leader: eventing-nats-1
+  - Replicas: 1
+
+**State after test run:**
+- Stream:
+  - Leader: eventing-nats-0
+  - Replicas: 1
+
+**Test Results:**
+```
+-> Total Events Sent      by **Test Sender** : 85,876 (+ 19 (Failed) = 85,895)
+-> Total Events Received  by **Sink**        : 85,878 (means 2 events were duplicates)
+```
+
+**Test Results Dashboard:**
+
+![](assets/ha_crash1_1.png "")
+
+**Test Sender Stats:**
+
+![](assets/ha_crash1_2.png "")
+
+**Finding:** 
+
+If the stream leader NATS pod is deleted then NATS stops accepting messages for that stream until a new leader is elected. Therefore, event-publisher-proxy fails to publish messages until the leader came back.
+
+---
+
+## Test Scenario 4: K8s Node deleted during test where stream leader NATS Pod was deployed (with Stream Replicas: 3)
+
+The number of stream replicas was changed to 3 replicas in order to check the behaviour of single-node failure with multiple stream replicas.
```

Comment:
Feedback: It makes it really easier to read for me compared to a note 😃👍

Reply:
added intro sentence for each test scenario